### ISSUE TITLE
CoreWebView2.WebResourceResponseReceived for frames

### ISSUE THREAD

**Issue by perelmanari** [MicrosoftEdge/WebView2Feedback#3540]
The WebResourceRequested event was added relatively recently to report resource requests from frames by using: CoreWebView2.AddWebResourceRequestedFilter("*", CoreWebView2WebResourceContext.All, CoreWebView2WebResourceRequestSourceKinds.Document);

However, the corresponding WebResourceResponseReceived event is not raised. Can this please be added?

Thank you

**Comment by yildirimcagri** [comment]
Hi @perelmanari , are you asking for the WebResourceRequestedFilter to also apply for the WebResourceResponseReceived event?

**Comment by perelmanari** [comment]
Yes. Either that or to add a WebResourceResponseFilter method. I am just looking for a way to get the WebResourceResponseReceived event.

**Comment by huoyaoyuan** [comment]
I'm also hitting this.

Currently, unlike WebResourceRequested, the WebResourceResponseReceived isn't applicable for the filter. However, the request inside frames aren't raised for events.
